### PR TITLE
Feature/ys apply yourself to workshop #1541

### PR DESCRIPTION
**Pull request by sivxkxn**

### CODE REVIEW

File: src/app/header/header.component.ts [hunk 1]
```diff
@@ -116,8 +116,8 @@ export class HeaderComponent implements OnInit, OnDestroy {
   }
 
   private getFullName(user: User): string {
-    return `${user.lastName} ${user.firstName.slice(0, 1)}.${
-      user.middleName ? user.middleName.slice(0, 1) + '.' : ''
+    return `${user?.lastName} ${user?.firstName.slice(0, 1)}.${
```
Comment (anastasiiaderkach1996) [inline src/app/header/header.component.ts:119]: It would be better to made one check instead four.

Comment (sivxkxn) [reply src/app/header/header.component.ts]: Removed this changes completely

File: src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/child-info-box/child-info-box.component.html [hunk 2]
```diff
@@ -15,12 +15,12 @@ <h5 class="text">Стать:</h5>
...
     </div>
 
-    <div class="info-group">
+    <div class="info-group" *ngIf="!child.isParent">
       <h5 class="text">Соціальна група:</h5>
       <p>{{ child.socialGroups.join(',') }}</p>
     </div>
 
-    <div class="info-group">
+    <div class="info-group" *ngIf="!child.isParent">
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/child-info-box/child-info-box.component.html:23]: I think we can hide it under one ngIf

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/child-info-box/child-info-box.component.html (hunk 2 above)
Comment (sivxkxn) [reply src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/child-info-box/child-info-box.component.html]: done

**Review by litvinets — APPROVED** [review]

**Pull request merged**